setup.py

```diff
@@ -18,6 +18,18 @@
         "gdc-ng-models @ git+https://github.com/NCI-GDC/gdc-ng-models.git@1.6.4#egg=gdc-ng-models",
         "psqlgraph @ git+https://github.com/NCI-GDC/psqlgraph.git@4.0.1#egg=psqlgraph",
     ],
+    extras_require={
+        "dev": [
+            "dataclasses; python_version < '3.7'",
+            "notebook",
+            "jupyter",
+            "jupyter-client",
+            "jupyter-console",
+            "jupyter-core",
```

Comment:
Do we really need all these for "dev" (and testing)?
Maybe add one more,  "jupyter"?

Another alternative would be to add "test" and only include pytest-related deps. This way, tox would install ".[test]" (assuming tests don't need jupyter stuff) and save some time.

It's ok as is. It's just that "dev" with everything will add bootstrap time to each tox build (each commit).